pokemongo_bot/__init__.py

```diff
@@ -34,10 +34,15 @@ def start(self):
 
     def take_step(self):
         location = self.navigator.take_step()
-        cells = self.find_close_cells(*location[0:2])
+        self.work_cells(1, self.position[0:2])
```

Comment:
api._position_lng and api._position_lng are integers. Almost all methods of PokemonGoBot and some dependencies expect float.  What to do?